lib/std/base64.zig

```diff
@@ -1,3 +1,34 @@
+//! This module provides functions for encoding binary data to
+//! printable ASCII characters and decoding such encodings back to
+//! binary data.
+//!
+//! The encodings are described in RFC 4648.
+//!
+//! There are four variants defined here:
+//!
+//! standard:
```

Comment:
why waste space here? You can make it very dense without losing readability, ie
```txt
1. standard:         Standard Base64 codecs + padding
2. standard_no_pad:  Standard Base64 codecs - padding
3. url_safe:         URL-safe Base64 codecs + padding
4. url_safe_no_pad:  URL-safe Base64 codecs - padding
```
or use `with` and `without`